content/20.treatments.md

```diff
@@ -558,24 +489,22 @@ One trial is currently investigating the utility of vitamin D as an immune-modul
 Other trials are also underway examining various factors including mortality, symptom recovery, severity of disease, rates of ventilation, inflammatory markers such as C-reactive protein (CRP) and IL-6, blood cell counts, and the prophylactic capacity of vitamin D administration [@clinicaltrials:NCT04385940; @clinicaltrials:NCT04334005; @clinicaltrials:NCT04411446; @clinicaltrials:NCT04344041].
 Concomitant administration of vitamin D with pharmaceuticals such as aspirin [@clinicaltrials:NCT04363840] and bioactive molecules such as resveratrol [@clinicaltrials:NCT04400890] are also under investigation.
 
-**Summary.**
-Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin D status warrants further investigation.
-This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world.
-This measure is likely to limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D, potentially weakening the immune system and increasing the risk of COVID-19 infection.
+Whether or not vitamin D status is linked to COVID-19 outcomes, even spuriously, thus remains open for debate.
```

Comment:
```suggestion
While there is increasing evidence that vitamin D status is linked to COVID-19 outcomes, the effectiveness of its supplementation remains open for debate.
```

Reply:
I will add the studies below to support this statement in text shortly. It is the only one I would tone down too much as there is a lot of research now supporting vit D investigation. 

https://doi.org/10.3390/nu12082358
https://doi.org/10.1371/journal.pone.0239799
https://doi.org/10.1111/febs.15495
https://doi.org/10.1111/cen.14276

Reply:
Studies added to this section, requires a re-read and see if you are happy with the tone :) @rando2